### ISSUE TITLE
[Exploit] Unlimited Mana on my person

### ISSUE THREAD

**Issue by itanshi** [Vazkii/Botania#654]
I put on the (empty) Greater Band of Mana, Greater Band of Aura, Flugal Tiara/Tainted Blood Pendant with Regen 2. I added a point (or more) mana to the GBoM and flew. I ran out of mana, I did not lose flight. Likewise, I can have constant regen 2 for free. I saw this in version 157 and just confirmed it in 169.

edit: I m doing tests with botania alone. So far it works on .6.5 regrowth and .7.0 regrowth

**Comment by Vazkii** [comment]
Can't reproduce.

**Comment by itanshi** [comment]
https://www.youtube.com/watch?v=kBhiN1-tCS0&feature=youtu.be reproduced and filmed

**Comment by Vazkii** [comment]
Still can't reproduce it. Does it happen on singleplayer?

**Comment by itanshi** [comment]
I'm having a hard time reproducing it in single/lan mode. I'll keep trying.

**Comment by Vazkii** [comment]
Are you running a cauldron server? From the looks of the chat in your video it looked like it.

**Comment by itanshi** [comment]
It is forge

**Comment by itanshi** [comment]
I have confirmed this still works on multiplayer version 176 of botania

**Comment by Lazersmoke** [comment]
@itanshi version 176 is very outdated now, you should try to reproduce it on the latest botania (190 ATM)

**Comment by itanshi** [comment]
I confirm this on version 208 multiplayer server

**Comment by itanshi** [comment]
I confirm this on 210. I hugged a cactus with regen 2 amulet and no mana. I healed completely and hugged it again.

**Comment by itanshi** [comment]
Ok I just tested with Multiplayer Botania 219. i decided to add the tinker hearts. All I have in bauble slot is greater band of mana, regen 2 Tainted amulet and greater band of aura. I was able to fully regen the hearts while the durability of the greater band of mana remained at 0.

I stood on a cactus and the regen effect turned on and off, which I think its normal behavior. Could the greater band of aura give mana at a rate to maintain (constant-ish) regen 2 with constant damage from 0? Its not enough to keep me alive, but it keeps trying to regen me despite starting at 0 mana.
